# 
# Submitted-By: Robert Schwebel, 2005-03-13
# Committed-By: Robert Schwebel
#
# Error: 
#
# MTD utilities forget to use the preprocessor flags; as we feed in
# special include directories via -isystem in PTXdist this is not used
# and crashes. 
#
# Description: 
#
# no public discussion about this topic yet. 
#
# State:
# 
# unknown
#

diff -urN mtd-20040621-1/util/Makefile mtd-20040621-1-cppflags/util/Makefile
--- mtd-20040621-1/util/Makefile	2004-05-28 12:51:10.000000000 +0200
+++ mtd-20040621-1-cppflags/util/Makefile	2005-03-13 15:58:23.000000000 +0100
@@ -21,7 +21,7 @@
 	$(CC) $(LDFLAGS) -g -o $@ $^
 
 %.o: %.c
-	$(CC) $(CFLAGS) -g -c -o $@ $< -g -Wp,-MD,.$<.dep
+	$(CC) $(CFLAGS) $(CPPFLAGS) -g -c -o $@ $< -g -Wp,-MD,.$<.dep
 
 .SUFFIXES:
 
